10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
 
 \section{Sys/sys}
 \label{std-sys}
+
+\section{Remoting}
+\label{std-remoting}
+
+Haxe remoting is a way to talk between different platforms. With Haxe remoting we can communicate transparently, send data and call methods, between server and client side.
+
+\subsection{Remoting Connection}
+\label{std-remoting-connection}
+
+In order to use remoting, there must be a connection established first. There are two kinds of Haxe Remoting connections: 
+\begin{description}
+	\item[\expr{haxe.remoting.Connection}] is used for \emph{synchronous connections}, where the results can be directly obtained when calling a method. 
+	\item[\expr{haxe.remoting.AsyncConnection}] is used for \emph{asynchronous connections}, where the results are events that will happen later in the execution process.
```

Comment:
consider "process of execution" or "execution of the proces..." I think the first.